gravatar/src/main/java/com/gravatar/restapi/models/VerifiedAccount.kt

```diff
@@ -32,16 +33,20 @@ public class VerifiedAccount private constructor(
     // The URL to the user's profile on the service.
     @Json(name = "url")
     public val url: java.net.URI,
+    // Whether the verified account is hidden from the user's profile.
+    @Json(name = "is_hidden")
+    public val isHidden: kotlin.Boolean,
```

Comment:
We probably need to update the components to filter those hidden ones, right?

Reply:
I mean in a separate PR ofc!

Reply:
Yes, we need to do that. Ticket created-> #356